.ado/publish.yml

```diff
@@ -84,10 +90,15 @@ jobs:
       vmImage: vs2017-win2016
 
     steps:
-      # Sync to point where the version numbers have been updated
+      - task: DownloadBuildArtifacts@0
+        displayName: Download VersionEnvVars
+        inputs:
+          artifactName: VersionEnvVars
+          downloadPath: $(Build.StagingDirectory)
+
       - task: CmdLine@2
         inputs:
-          script: git checkout $(publishCommitId)
```

Comment:
>git checkout  [](start = 18, length = 13)

Not needed anymore?

Reply:
No, the versionEnvVars script gets all the info we need.